src/generic_ui/polymer/copypaste.html

```diff
@@ -43,6 +46,14 @@
       <span hidden?='{{ SharingState.NONE === ui.copyPasteSharingState }}'>
         Share a one time connection
       </span>
+      <span hidden?='{{ ui.copyPasteSharingState !== SharingState.NONE || ui.copyPasteGettingState !== GettingState.NONE }}'>
+        Manual Connection
```

Comment:
Not sure I understand what manual connection means here.. Can you attach a screenshot for this change?